scripts/zones/Phomiuna_Aqueducts/npcs/_0rx.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+--    Area: Phomiuna_Aqueducts
+--  NPC: Oil Lamp - Water (East)
+-- ID: 16888080  !pos 104 -26 77
+-----------------------------------
+require("scripts/globals/missions");
```

Comment:
These mission requires aren't being used for these lamps...!

Reply:
Mission requires deleted. They were copied/pasted from west side lamps.